pyccl/tracers.py

```diff
@@ -292,6 +298,9 @@ class NumberCountsTracer(Tracer):
     def __init__(self, cosmo, has_rsd, dndz, bias, mag_bias=None):
         self._trc = []
 
+        # we need the distance functions at the C layer
+        cosmo.compute_distances()
```

Comment:
Why do you initialize this explicitly here? I thought you were doing that in the kernel calls already.

Reply:
It depends on the ordering of the calls. This call is a no-op if the init has already been done, so no harm here.